src/AnswerButton.css

```diff
@@ -1,11 +1,87 @@
+.answer-buttons-container  {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows:  7;
+    grid-column-gap: 20px;
+    grid-row-gap: 10px;
+}
+
+.answer-button {
+    border-radius: 1000px; /* any large number of pixels gets the desired effect */
```

Comment:
It feels weird to use pixel numbers here, but if I use percentages, it gives me an oval shaped button,  which is not what I'm aiming for!  Is there a better way to do this?

Reply:
Hm, no, I think this is actually the correct way to do this, or hard code the actual exact value.